Quarks/test_quark.py

```diff
@@ -4,20 +4,19 @@
 t = Quark()
 
 def test_quark_1():
-    assert if self.flavor == "up":
-            self.flavor = "down"
+    assert t.flip("up") == "down"
```

Comment:
A way that works is the following,

t.flavor = "up"
t.flip()
assert t.flavor == "down"

This passes and tests the method.